docs/reference/codelists.md

```diff
@@ -204,9 +348,25 @@ file: ../../codelists/closed/hazard_type.csv
 ---
 ```
 
+### impact_type
+
+This codelist is referenced by the following properties:
+
+- `Impact/type`
+
+This codelist has the following codes:
+
+```{csv-table-no-translate}
+---
+header-rows: 1
+widths: auto
+file: ../../codelists/closed/impact_type.csv
+---
+```
+
 ### process_type
 
```

Comment:
No 'This code is referened by' section here?

Reply:
See https://github.com/GFDRR/rdl-standard/pull/124#discussion_r1255006311